SchemaInventory.json

```diff
@@ -394,6 +394,20 @@
       "date": "Unknown",
       "dynamic": "No",
       "approved": "Yes"
+    },
```

Comment:
The ECDbMap 2.0.1 schema needs to have it's inventory updated, it used to be listed as known bad because it would force the profile upgrade back when we did not want to force the 4.0.0.2 ECDb profile upgrade.

Additionally the schemas should be marked as approved and verfied.  YOu can use me as the verifier, I reviewed the changes to all three schemas and they look good to me.

Reply:
The inventory now have the updated entry for ECDbMap 2.0.1. Additionally, I have also updated the `verified` and `approved` values for all these 3 schemas. Previously, I intentionally skipped them to get your review first.